nixos/modules/services/web-servers/nginx/default.nix

```
@@ -446,6 +446,24 @@ in
         ";
       };
 
+      defaultSSLPort = mkOption {
+        type = types.port;
+        default = 443;
+        example = literalExpression ''443'';
+        description = "
+          Default SSL port for the server
+        ";
+      };
+
+      defaultPort = mkOption {
+        type = types.port;
+        default = 80;
+        example = literalExpression ''80'';
+        description = "
+          Default port for the server
```

Perhaps this wording helps make it beyond-obvious by contrasting with "SSL" in the docs for `defaultSSLPort`?
```suggestion
          Default HTTP port for the server
```